avalancheup-aws/src/apply/mod.rs

```diff
@@ -67,25 +117,418 @@ pub fn command() -> Command {
+                .long("vm-binary-local-path")
+                .help("VM binary local file path")
+                .required(true)
+                .num_args(1),
+        )
+        .arg(
+            Arg::new("VM_BINARY_S3_KEY")
+                .long("vm-binary-s3-key")
+                .help("Sets the S3 key for the Vm binary (if empty, default to local file name)")
+                .required(false)
+                .num_args(1),
+        )
+        .arg(
+            Arg::new("VM_BINARY_REMOTE_DIR")
+                .long("vm-binary-remote-dir")
+                .help("Plugin dir for VM binaries")
+                .required(true)
+                .num_args(1),
+        )
+        .arg(
+            Arg::new("VM_ID")
+                .long("vm-id")
+                .help("Sets the 32-byte Vm Id for the Vm binary (if empty, converts chain name to Id)")
+                .required(false)
+                .num_args(1),
+        )
+        .arg(
+            Arg::new("CHAIN_NAME")
+                .long("chain-name")
+                .help("Sets the chain name")
+                .required(true)
+                .num_args(1),
+        )
+        .arg(
+            Arg::new("CHAIN_GENESIS_PATH")
+                .long("chain-genesis-path")
+                .help("Chain genesis file path")
+                .required(true)
+                .num_args(1),
+        )
+        .arg(
+            Arg::new("CHAIN_CONFIG_LOCAL_PATH")
+                .long("chain-config-local-path")
+                .help("Chain configuration local file path")
+                .required(false)
+                .num_args(1),
+        )
+        .arg(
+            Arg::new("CHAIN_CONFIG_S3_KEY")
+                .long("chain-config-s3-key")
+                .help("Sets the S3 key for the subnet chain config (if empty, default to local file name)")
+                .required(false)
+                .default_value("subnet-chain-config.json")
+                .num_args(1),
+        )
+        .arg(
+            Arg::new("CHAIN_CONFIG_REMOTE_PATH")
+                .long("chain-config-remote-dir")
+                .help("Chain configuration remote file path")
+                .required(false)
+                .num_args(1),
+        )
+        .arg(
+            Arg::new("AVALANCHEGO_CONFIG_REMOTE_PATH")
+                .long("avalanchego-config-remote-path")
+                .help("avalanchego config remote file path")
+                .required(true)
+                .num_args(1),
+        )
+        .arg(
+            Arg::new("SSM_DOCS")
+                .long("ssm-docs")
+                .help("Sets the hash map of AWS region to SSM document name for subnet and chain install (see avalanche-ops/src/aws/cfn-templates/ssm_install_subnet_chain.yaml)")
+                .required(false)
+                .value_parser(HashMapStringToStringParser {})
+                .num_args(1),
+        )
+        .arg(
+            Arg::new("TARGET_NODES")
```

Comment:
how would the user know the target nodes if they are created in the `apply` step?